Source/KakapoDB.swift

```diff
@@ -63,6 +69,14 @@ public class KakapoDB {
         return object!
     }
     
+    /**
+     Inserts Storable objects based on their default initializer
+     
+     - parameter (unamed): The Storable Type to be created
+     - parameter number: The number of elements to create, defaults to 1
+     
+     - returns: An array containing the new inserted Storable objects
+     */
     public func create<T: Storable>(_: T.Type, number: Int = 1) -> [T] {
         let ids = barrierSync {
```

Comment:
I'm wondering why we don't call `insert` for create, to avoid duplicate/similar code